src/main/java/codeu/controller/LogoutServlet.java

```diff
@@ -19,22 +19,26 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
 
 /** Servlet class responsible for the logout page. */
 public class LogoutServlet extends HttpServlet {
-
-  /**
-   * This function fires when a user requests the /login URL. It simply forwards the request to
-   * login.jsp.
-   */
+  /** Set up state for handling profile pages. */
+  @Override
+  public void init() throws ServletException {
```

Comment:
Why do you need this? I think you can leave it out since all you are doing is calling the base classes method with the same name.